configs/mainnet.json

```diff
@@ -483,5 +531,127 @@
       "label_denom": "brdg/a02afc2c1edf77cc023eefa25fc036c184612faf9365cda9c1daa3b1675ebf8f",
       "target_denom": "brdg/a02afc2c1edf77cc023eefa25fc036c184612faf9365cda9c1daa3b1675ebf8f"
     }
-  ]
+  ],
+  "blockchain_gas_infos": {
+    "Archway": {
+      "fee_denom": "ibc/652218B355FBD814037EEF28505F3D91F0489325A64FA5D865EF425DD59317BC",
+      "decimals": 18
+    },
+    "Axelar": {
+      "fee_denom": "ibc/062588846168EDFECF20F7681F2A0A131A53B9A3C8535BCA8F7D1A268246974B",
+      "decimals": 6
+    },
+    "DYDX": {
+      "fee_denom": "ibc/9DC57FF60E125F9B976EB0DF4F2122C580AD1DB8E27BA71E6D2C1AA56A5FD312",
+      "decimals": 18
+    },
+    "Osmosis": {
+      "fee_denom": "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518",
+      "decimals": 6
+    },
+    "Cosmos Hub": {
+      "fee_denom": "ibc/A4DB47A9D3CF9A068D454513891B526702455D3EF08FB9EB558C561F9DC2B701",
+      "decimals": 6
+    },
+    "Kujira": {
+      "fee_denom": "ibc/662914D0C1CEBCB070C68F061D035E8B10A07C79AB286E7342C85F3BE74612C5",
+      "decimals": 6
+    },
+    "Noble": {
+      "fee_denom": "usdc.1.6.53ff75",
+      "decimals": 6
+    },
+    "Stargaze": {
+      "fee_denom": "ibc/C688EBE80ECC20865331A18D8D19654754F6B189977715FED00FBA2BE41F144F",
+      "decimals": 6
+    },
+    "Stride": {
+      "fee_denom": "ibc/3552CECB7BCE1891DB6070D37EC6E954C972B1400141308FCD85FD148BD06DE5",
+      "decimals": 18
+    },
+    "Terra": {
+      "fee_denom": "ibc/2B58B8C147E8718EECCB3713271DF46DEE8A3A00A27242628604E31C2F370EF5",
+      "decimals": 6
+    },
+    "Mantle": {
+      "fee_denom": "bit.1.2.9d68c4",
```

Comment:
Think the fee denom for Mantle is `brdg/a02afc2c1edf77cc023eefa25fc036c184612faf9365cda9c1daa3b1675ebf8f`